src/tools/policy2proto.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {mapToDictionary} from '../runtime/util.js';
+import {PolicyConfig, PolicyField, PolicyTarget, Policy, PolicyAllowedUsageType} from '../runtime/policy/policy.js';
+import {AnnotationRef} from '../runtime/recipe/annotation.js';
+import {PolicyRetentionMediumEnum, PolicyFieldUsageEnum, PolicyEgressEnum} from './manifest-proto.js';
+
+export function policyToProtoPayload(policy: Policy) {
```

Comment:
this is a one-way conversion, always manifest object to proto, but never the other way around, right?

Reply:
That's right.